vispy/app/backends/_qt.py

```diff
@@ -125,55 +138,107 @@ def _check_imports(lib):
 #   Shift  50, 62
 #   Ctrl   37, 105
 #   Alt    64, 108
-KEYMAP = {
-    QtCore.Qt.Key_Shift: keys.SHIFT,
-    QtCore.Qt.Key_Control: keys.CONTROL,
-    QtCore.Qt.Key_Alt: keys.ALT,
-    QtCore.Qt.Key_AltGr: keys.ALT,
-    QtCore.Qt.Key_Meta: keys.META,
-
-    QtCore.Qt.Key_Left: keys.LEFT,
-    QtCore.Qt.Key_Up: keys.UP,
-    QtCore.Qt.Key_Right: keys.RIGHT,
-    QtCore.Qt.Key_Down: keys.DOWN,
-    QtCore.Qt.Key_PageUp: keys.PAGEUP,
-    QtCore.Qt.Key_PageDown: keys.PAGEDOWN,
-
-    QtCore.Qt.Key_Insert: keys.INSERT,
-    QtCore.Qt.Key_Delete: keys.DELETE,
-    QtCore.Qt.Key_Home: keys.HOME,
-    QtCore.Qt.Key_End: keys.END,
-
-    QtCore.Qt.Key_Escape: keys.ESCAPE,
-    QtCore.Qt.Key_Backspace: keys.BACKSPACE,
-
-    QtCore.Qt.Key_F1: keys.F1,
-    QtCore.Qt.Key_F2: keys.F2,
-    QtCore.Qt.Key_F3: keys.F3,
-    QtCore.Qt.Key_F4: keys.F4,
-    QtCore.Qt.Key_F5: keys.F5,
-    QtCore.Qt.Key_F6: keys.F6,
-    QtCore.Qt.Key_F7: keys.F7,
-    QtCore.Qt.Key_F8: keys.F8,
-    QtCore.Qt.Key_F9: keys.F9,
-    QtCore.Qt.Key_F10: keys.F10,
-    QtCore.Qt.Key_F11: keys.F11,
-    QtCore.Qt.Key_F12: keys.F12,
-
-    QtCore.Qt.Key_Space: keys.SPACE,
-    QtCore.Qt.Key_Enter: keys.ENTER,
-    QtCore.Qt.Key_Return: keys.ENTER,
-    QtCore.Qt.Key_Tab: keys.TAB,
-}
-BUTTONMAP = {0: 0, 1: 1, 2: 2, 4: 3, 8: 4, 16: 5}
+if qt_lib == 'pyqt6':
```

Comment:
Could this be simplified by doing something like `qt_keys = QtCore.Qt.Key if qt_lib == 'pyqt6' else QtCore.Qt` and then update the dictionary to use `qt_keys` instead?

Reply:
I agree, I thought about that but I didn't want to make too many changes and depart from what people are used to see. I know that I don't like seeing too many changes in a big project :)
Will do.